packages/core/src/Stores/client-store.js

```diff
@@ -170,6 +170,7 @@ export default class ClientStore extends BaseStore {
     subscriptions = {};
     exchange_rates = {};
     client_kyc_status = {};
+    isLoggedOutV2 = false;
```

Comment:
Hmm, is there any other way how we can do without implementing this variable? I just find it odd to have a variable specifically for log out state of V2. Can we instead reset everything once the new account is opened?